article-submission-example/build.gradle

```diff
@@ -36,13 +42,21 @@ repositories {
     maven {
         url = 'https://maven.northconcepts.com/public/repositories/templatemaster'
     }
+        maven { url 'https://jitpack.io' }
```

Comment:
Can we add `content` sections like we do with the `repository.apache.org` repo below?

Reply:
Actually we no longer need this, removed it

Reply:
Done